packages/discord.js/src/client/websocket/handlers/THREAD_UPDATE.js

```diff
@@ -11,6 +11,6 @@ module.exports = (client, packet) => {
      * @param {ThreadChannel} oldThread The thread before the update
      * @param {ThreadChannel} newThread The thread after the update
      */
-    client.emit(Events.THREAD_UPDATE, old, updated);
+    client.emit(Events.ThreadCreate, old, updated);
```

Comment:
```suggestion
    client.emit(Events.ThreadUpdate, old, updated);
```
Isn't it supposed to be `ThreadUpdate`?